desktop/components/main_layout/header/view.coffee

```diff
@@ -38,6 +39,8 @@ module.exports = class HeaderView extends Backbone.View
       mode: 'suggest'
       limit: 7
 
+    splitTest('highlight_header_search').view()
```

Comment:
I'm ok, it doesnt even need to be near where the search bar was instantiated. Essentially any page that shows the main header should fire this, so this feels very 👌 and appropriate to me.